server-spi/src/main/java/org/keycloak/models/KeycloakUriInfo.java

```diff
@@ -81,7 +82,7 @@ public UriBuilder getAbsolutePathBuilder() {
     @Override
     public URI getBaseUri() {
         if (baseURI == null) {
-            baseURI = delegate.getBaseUriBuilder().scheme(scheme).host(hostname).port(port).build();
+            baseURI = delegate.getBaseUriBuilder().scheme(scheme).host(hostname).port(port).replacePath(contextPath).build();
```

Comment:
Should this change be applied  to other methods of this class as well (`getAbsolutePath`, `getRequestUri`)?

Reply:
Added